packages/appium/README.md

```diff
@@ -1,50 +1,212 @@
 ## Appium
```

Comment:
is this file a duplicate of the one we have in the root folder? can we just link it instead of having a duplicate?

Reply:
Yes, it's a duplicate, and no, we can't link it.  The purpose is so that it's published on the package page on npmjs.com

Reply:
would it be possible to somehow automate sync between these two files? it would be a pain to manually keep them in sync every time the main readme is changed

Reply:
yes, that's already set up (in theory)

the `appium` one is kept in sync with the root `README.md`